bin/index.ts

```diff
@@ -3,5 +3,9 @@
 import { cli } from '../lib/cli';
 import { getDefaultConfigPath, readConfigFromFile } from '../lib/config';
 
-const config = readConfigFromFile(getDefaultConfigPath());
-cli(config);
+try {
+  const config = readConfigFromFile(getDefaultConfigPath());
+  cli(config);
+} catch {
```

Comment:
Do you think we could handle the error and print the message? I think it would be better to avoid silencing other possible errors.